main/src/ca/uwaterloo/flix/language/phase/jvm/BackendType.scala

```diff
@@ -134,4 +135,32 @@ object BackendType {
          MonoType.SchemaEmpty | MonoType.SchemaExtend(_, _, _) | MonoType.Native(_) |
          MonoType.Region => BackendObjType.JavaObject.toTpe
   }
+
+  /**
+    * Computes the `BackendType` based on the given `MonoType`.
+    */
+  def toBackendType(tpe: MonoType): BackendType = tpe match {
```

Comment:
This performs some erasure but doesn't say so in its name or docs. Maybe call it `toFlixErasedBackendType` and then explain what that means.